spec/node_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'node'
+
+describe Node do
+  subject(:node) { Node.new }
```

Comment:
Pro tip:

You can use a short cute in the creation of subject.
```
subject(:my_stuff) { described_class.new }
```
Where described_class is equal to the name of the spec. So this is very useful if you want to create specs that share behaviour, check this link:

https://www.relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples